### ISSUE TITLE
2.0.0-M.3 swagger

### ISSUE THREAD

**Issue by anton-johansson** [stoicflame/enunciate#130]
If I DON'T have ```<swagger disabled="true" />```  in my ```enunciate.xml```, I get a nasty exception when running ```mvn enunciate:docs```

```
[INFO] Scanning for projects...
[INFO] 
[INFO] Using the builder org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder with a thread count of 1
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building Viskan Distanshandel System AB :: API - Application Interface - CXF Connector 0.0.0.Dev-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- enunciate-maven-plugin:2.0.0-M.3:docs (default-cli) @ <artifactId> ---
[INFO] [ENUNCIATE] Using enunciate configuration at C:\Users\<username>\Projects\<project>\enunciate.xml
[INFO] Reflections took 1104 ms to scan 38 urls, producing 658 keys and 658 values 
[WARNING] [ENUNCIATE] Error processing C:\Users\<username>\Projects\<project>\target\site\apidocs\ui\swagger.json.
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 12.337 s
[INFO] Finished at: 2015-09-05T23:27:27+01:00
[INFO] Final Memory: 46M/669M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.webcohesion.enunciate:enunciate-maven-plugin:2.0.0-M.3:docs (default-cli) on project <artifactId>: Execution default-cli of goal com.webcohesion.enunciate:enunciate-maven-plugin:2.0.0-M.3:docs failed: com.webcohesion.enunciate.EnunciateException: com.fasterxml.jackson.core.JsonParseException: Unexpected character ('C' (code 67)): was expecting comma to separate OBJECT entries
[ERROR] at [Source: java.io.FileReader@3b4444eb; line: 688, column: 57]: OnError while emitting onNext value: com.webcohesion.enunciate.EnunciateContext.class
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
```

**Comment by stoicflame** [comment]
Will you please find "swagger.json" in the target directory and tell me what line 688 looks like? Maybe include lines 680-700 for context?

**Comment by anton-johansson** [comment]
Sure, I'll do it as soon as I have time to spare!

**Comment by anton-johansson** [comment]
```json
	"json_addressupdate" : {
		"type" : "object",
		"title" : "AddressUpdate",
		"properties" : {
			"address" : {
				"description" : "The address.",
				"type" : "string"
			},
			"address_desc" : {
				"description" : "The address description, for example "C/o Anton Johansson".",
				"type" : "string"
			},
			"city" : {
				"description" : "The city of the address.",
				"type" : "string"
			}
		}
	}
```

I don't think I even need to say which line is 688. :) Seems like the description strings aren't escaped.

**Comment by stoicflame** [comment]
Thanks. Fixed at ffa4ae4.

**Comment by stoicflame** [comment]
Fix was included in [Milestone 4](https://github.com/stoicflame/enunciate/releases/tag/v2.0.0-M.4). Thanks for the report, and please report any new issues.
